pdns/capabilities.cc

```diff
@@ -27,12 +27,13 @@
 
 #ifdef HAVE_LIBCAP
 #include <sys/capability.h>
+#include <sys/prctl.h>
 #endif
 
 #include "capabilities.hh"
 #include "misc.hh"
 
-void dropCapabilities(std::set<std::string> capabilitiesToKeep)
+bool dropCapabilities(std::set<std::string> capabilitiesToKeep)
```

Comment:
we don't consume this bool anywhere, unless I'm missing something

Reply:
True, I wanted to stop dnsdist from warning about not being able to drop our capabilities when we don't have any.
But I still wanted to provide the caller with the information of whether something was actually done, even though I don't really care in dnsdist.

Reply:
Oh, right, the `throw` is masked by `return false` for the EPERM case. Ok!